tests/listen_and_simulate.rs

```diff
@@ -1,12 +1,9 @@
-extern crate rdev;
-extern crate tokio;
-use rdev::{simulate, EventType, Key};
-use std::time::Duration;
 use lazy_static::lazy_static;
-use rdev::{listen, Event};
-use std::thread;
-use std::sync::Mutex;
+use rdev::{listen, simulate, Event, EventType, Key};
```

Comment:
We should remove tokio from dependencies too no ?

Reply:
If you mean from cargo.toml, yep, I did.